css/component_styles/getStarted.css

```diff
@@ -0,0 +1,89 @@
+#components {
+    margin-top: 5rem;
+}
+
+/* drawer css */
+.drawer {
+    background-color: var(--secondary-color);
+    padding: 1rem;
+}
+
+.drawer ul {
+    display: flex;
+    column-gap: 1rem;
+    overflow: auto;
+}
+
+::-webkit-scrollbar {
+    display: none;
+}
+
+.drawer ul li {
+    padding: 0.5rem;
+    font-weight: bold;
+    border-radius: 0.5rem;
+}
+
+.active,
+.drawer ul li:hover {
+    cursor: pointer;
+    background-color: var(--hover-color);
+    color: var(--primary-color);
+}
+
+.horizontal-line {
+    display: none;
+}
+
+/* content section */
+
+.content {
+    width: 90%;
+    margin: 1rem auto;
+}
+
+.content h4, .content h5 {
+    color: var(--primary-color);
+    margin-bottom: 1rem;
+}
+
+.code {
+    color: var(--primary-color);
+    background-color: var(--hover-color);
+    padding: 0.2rem;
+    border-radius: 0.2rem;
+}
+
+.code-img {
+    width: 80%;
+    height: auto;
+    margin: 2rem auto;
+}
+
+.horizontal-line2{
+    display: block;
+    border-bottom: solid 0.1rem var(--secondary-color);
+    width: 100%;
+    padding: 0 !important;
```

Comment:
good to not have !important unless it's the last resort

Reply:
thanks for the correction, i did removed !important and fixed the issue